1.0/docs/devguide/styling.md

```diff
@@ -241,8 +241,9 @@ doesn't set the custom property:
 
 Thus, custom CSS properties introduce a powerful way for element authors to
 expose a theming API to their users in a way that naturally fits right alongside
-normal CSS styling. It is,already on a standards track with support in
-Firefox and planned support announced in Chrome and Safari.
+normal CSS styling. 
+
+It is already on a standards track (currently in the candidate recommendation (CR) stage of the process) with support in Firefox and Safari. It is currently available in Chrome as an experimental Web platform feature, among the features that need to be explicitly turned on by developers & enthusiasts. [Starting with Chrome version 49](https://www.chromestatus.com/features/6401356696911872), a version of Chrome currently in beta as of February 2016, custom CSS properties will be supported by default.
```

Comment:
@ebidel That's what I initially thought, but went initially with a more conservative route; I'll change that now.